src/Radar.js

```diff
@@ -0,0 +1,237 @@
+      defaultMouseMove(d, i);
+      const id = this._id(d, i);
+
+      this.hover((h, x) => this._x(h, x) === id);
+    };
+
+    this._radarPadding = 100;
+
+    this._shape = constant("Path");
+    this._shapeConfig = assign(this._shapeConfig, {
+      Circle: {
+        r: accessor("r", 0),
+        fill: constant("none"),
+        stroke: constant("#CCC"),
+        strokeWidth: constant(1)
+      },
+      Path: {
+        fillOpacity: 0.7
+      }
+    });
+    this._xConfig = {
+      height: 0
+    };
+    this._value = accessor("value");
+
+    this._y1 = false;
+  }
+
+  /**
+      Extends the draw behavior of the abstract Plot class.
+      @private
+  */
+  _draw(callback) {
+    const height = this._height - this._margin.top - this._margin.bottom,
+          width = this._width - this._margin.left - this._margin.right;
+
+    const radius = (Math.min(height, width) - this._radarPadding) / 2,
+          transform = `translate(${width / 2}, ${height / 2})`;
+
+    const maxValue = Math.max(...this._data.map((d, i) => this._value(d, i))),
+          nestedAxisData = nest()
+        .key(this._y)
+        .entries(this._data),
+          nestedGroupData = nest()
+        .key(this._x)
+        .entries(this._data);
+
+    const circularAxis = Array.from(Array(this._levels).keys()).map(d => ({
+      id: d,
+      r: radius * ((d + 1) / this._levels)
+    }));
+
+    this._shapes.push(
+      new Circle()
+        .data(circularAxis)
+        .select(
+          elem("g.d3plus-Radar-radial-circles", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .config(configPrep.bind(this)(this._shapeConfig, "shape", "Circle"))
+        .render()
+    );
+
+    const totalAxis = nestedAxisData.length;
+    const polarAxis = nestedAxisData
+      .map((d, i) => {
+        const angle = tau / totalAxis * i;
+        return {
+          id: d.key,
+          angle: 360 - 360 / totalAxis * i,
+          quadrant: parseInt(360 - 360 / totalAxis * i / 90, 10) % 4 + 1,
+          x: radius * Math.cos(angle),
+          y: radius * Math.sin(angle)
+        };
+      })
+      .sort((a, b) => a.key - b.key);
+
+    this._shapes.push(
+      new TextBox()
+        .data(polarAxis)
+        .rotate(
+          d => d.angle < 90 || d.angle > 270 ? -d.angle : -d.angle + 180
+        )
+        .rotateAnchor([0, 0])
+        .textAnchor(d => [1, 4].includes(d.quadrant) ? "start" : "end")
+        .height(14)
+        .text(d => d.id)
+        .x(d => d.x)
+        .y(d => d.y)
+        .select(
+          elem("g.d3plus-Radar-text", {
+            parent: this._select,
+            enter: {transform},
+            update: {transform}
+          }).node()
+        )
+        .render()
+    );
+
+    d3.selectAll("g.d3plus-Radar-text text")
+      .data(polarAxis)
+      .attr("x", d => d.x < 0 ? -10 : 10)
+      .attr("y", 5);
+
+    this._shapes.push(
```

Comment:
Only shapes that have user-data attached to them should be added to `this._shape`. Don't push the the axis lines. (this may be related to hover issues)